app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt

```diff
@@ -13,9 +14,10 @@ import com.kickstarter.services.DiscoveryParams
 import com.kickstarter.services.apiresponses.PushNotificationEnvelope
 import com.kickstarter.ui.data.*
 import com.kickstarter.ui.data.Mailbox
-import java.util.*
 import kotlin.collections.HashMap
 
+private const val CONTEXT_CTA = "context_cta"
```

Comment:
Suggestion here, move the constant to `EventContext.kt` as companion object 

```
    companion object {
        const val CONTEXT_CTA = "context_cta"
        fun getPledgeInitiateCTAProps(): HashMap<String, Any> = hashMapOf(CONTEXT_CTA to CtaContextName.PLEDGE_INITIATE.contextName)
    }
```

Reply:
So I purposely chose not to use a `companion object` because this isn't a global constant or method, and when you create a companion object, under the hood an unnecessary getter and setter are created to make these things accessible. Since these are not needed globally, it seems that this is unnecessary. 

https://stackoverflow.com/questions/44038721/constants-in-kotlin-whats-a-recommended-way-to-create-them